From c8fcf176688dab335b943a5e144925f646ce9bc4 Mon Sep 17 00:00:00 2001
From: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Date: Thu, 1 Jun 2023 16:47:15 +0200
Subject: [PATCH] arm64: defconfig: enable HASH support

Enable STM32 HASH support on 64 bits platform.

Signed-off-by: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Change-Id: I336d51666d660b27b283133415c682d76387d00e
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/310343
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/360631
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -5,6 +5,7 @@ CONFIG_CLKSRC_STM32_LP=y
 CONFIG_COUNTER=m
 CONFIG_CRYPTO_DEV_STM32_CRC=m
 CONFIG_CRYPTO_DEV_STM32_CRYP=m
+CONFIG_CRYPTO_DEV_STM32_HASH=m
 CONFIG_DRM=y
 CONFIG_DRM_BRIDGE=y
 CONFIG_DRM_DISPLAY_CONNECTOR=y
